sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1914,11 +2310,30 @@ class ReadFromBigQuery(PTransform):
         to create and delete tables within the given dataset. Dataset name
         should *not* start with the reserved prefix `beam_temp_dataset_`.
    """
+  class Method(object):
+    EXPORT = 'EXPORT'  #  This is currently the default.
+    DIRECT_READ = 'DIRECT_READ'
```

Comment:
nit: I believe this api are known as "BigQuery Storage Read API". Would it be better to specify method as `STORAGE_API_READ` or something similar not to confuse the user?

Reply:
I used to `DIRECT_READ` to keep it the same as the Java SDK. 

@kmjung: What do you think?

Reply:
I'm not sure that I have a sense of what Beam values here (consistency across languages vs. consistency with external naming). I guess my slight preference would be for the former, which would mean carrying over the `DIRECT_READ` name. @satybald @pabloem @chamikaramj what do you think?

Reply:
I also like keeping consistency with Java